payroll-processor-client/src/app/shared/env.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { Environment } from '../../environments/environment-types';
+import { environment } from '../../environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EnvService {
```

Comment:
This abstraction tends to stay pretty small in most apps, but we can use it for some utility access, like the `apiRootUrl` getter below.